### PR TITLE
Remove position: absolute when measuring fluid creative height.

### PR DESCRIPTION
**Pull request by glevitzky**
The presence of `position: absolute` on the slot element interferes with our ability to correctly measure the size of fluid creatives. Ad slots with fluid creatives resize themselves by taking up the maximum width available and setting their height to match the height of the creative's body element. This height is different if `position: absolute` is present (due to the element being removed from the normal document flow). So in the current way we measure and resize:

1) Add `i-amphtml-layout-awaiting-size` to slot element, which adds `position: absolute`
2) Measure height H from creative's body element
3) Set slot height to H
4) Remove `i-amphtml-layout-awaiting-size` from slot element, which removes `position: absolute`

H may be too large for the available width, since the element is returned to the normal document flow in step 4. The change here solves this by removing `position: absolute` right before step 2.

(This is a duplicate of #21944 which was closed due to some git-related weirdness.)

### CODE REVIEW

File: extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js [hunk 1]
```diff
@@ -1058,19 +1058,40 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
...
+  /**
+   * Sets the CSS 'position' property of this.element.
+   * @param {string} position The CSS position value.
+   * @return {!Promise} A promise that resolves when mutation is complete.
+   * @private
+   */
+  setCssPosition_(position) {
+    return this.measureMutateElement(
+        /** MEASURER */ () => {
+          this.getResource().measure();
```
Comment (jridgewell) [inline extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: Measure happens before the mutate. Is the intention here to change the style, then measure?

Comment (glevitzky) [reply extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: Come to think of it, I'm not sure if I need `this.getResource().measure()` -- it's just something I copied & pasted. But, yes, the idea is to change the style, and then _then_ read `this.iframe.contentWindow.document.body./*OK*/clientHeight`.

Comment (jridgewell) [reply extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: You should only need a `mutateElement`, then.

Comment (glevitzky) [reply extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: Done.

**Pull request merged**